.github/workflows/ruby.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: CI - PR/master
+
+on:
+  workflow_dispatch:
```

Comment:
No value here ?

Reply:
no, it means we can trigger the workflow manually, via a button. mostly useful for debugging stuff.